kujira/app/components/charts/bar-chart.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+import {
+    createRefreshingChart
+} from './chartsCommonContent';
+
+export default Ember.Component.extend({
+    height: null,
+    width: null,
```

Comment:
Height and width have wrong default values. Currently it is impossible to use this component without specifying them.